src/data/accessibility-quiz.ts

```diff
@@ -248,8 +248,17 @@ const accessibilityQuiz = [
     Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA"
   },
   {
-    Question:
-      "How can you make focus elements more accessible to screen readers on native mobile apps?",
+    Question: "What is the best way to make your website keyboard accessible?",
+    Answer: 
+    "Don't `<tabindex>` attribute greater than zero",
```

Comment:
we shouldn't use backticks here because the backticks will still show up in the final quiz
```suggestion
    "setting the tabindex attribute greater than zero",
```